### ISSUE TITLE
ReferenceError: unknown node of type "ChainExpression" with constructor "Object"

### ISSUE THREAD

**Issue by EvgenyOrekhov** [coderaiser/putout#70]
Reproduction:

```js
const bar = foo?.bar;
```

Full stack trace:

```text
ESLint: 7.31.0

ReferenceError: unknown node of type "ChainExpression" with constructor "Object"
Occurred while linting /example/src/App.ts:1
    at Generator.print (/example/node_modules/@babel/generator/lib/printer.js:258:13)
    at Generator.generate (/example/node_modules/@babel/generator/lib/printer.js:38:10)
    at Generator.generate (/example/node_modules/@babel/generator/lib/index.js:23:18)
    at generate (/example/node_modules/@babel/generator/lib/index.js:95:14)
    at NodePath.toString (/example/node_modules/@babel/traverse/lib/path/index.js:164:35)
    at const __identifier = __b (/example/node_modules/@putout/plugin-reuse-duplicate-init/lib/reuse-duplicate-init.js:17:57)
    at module.exports (/example/node_modules/try-catch/lib/try-catch.js:5:23)
    at /example/node_modules/@putout/engine-runner/lib/template/index.js:87:21
    at NodePath._call (/example/node_modules/@babel/traverse/lib/path/context.js:53:20)
    at NodePath.call (/example/node_modules/@babel/traverse/lib/path/context.js:40:17)
    at NodePath.visit (/example/node_modules/@babel/traverse/lib/path/context.js:90:31)
    at TraversalContext.visitQueue (/example/node_modules/@babel/traverse/lib/context.js:99:16)
    at TraversalContext.visitMultiple (/example/node_modules/@babel/traverse/lib/context.js:68:17)
    at TraversalContext.visit (/example/node_modules/@babel/traverse/lib/context.js:125:19)
    at Function.traverse.node (/example/node_modules/@babel/traverse/lib/index.js:76:17)
    at NodePath.visit (/example/node_modules/@babel/traverse/lib/path/context.js:97:18)
    at TraversalContext.visitQueue (/example/node_modules/@babel/traverse/lib/context.js:99:16)
    at TraversalContext.visitSingle (/example/node_modules/@babel/traverse/lib/context.js:73:19)
    at TraversalContext.visit (/example/node_modules/@babel/traverse/lib/context.js:127:19)
    at Function.traverse.node (/example/node_modules/@babel/traverse/lib/index.js:76:17)
    at traverse (/example/node_modules/@babel/traverse/lib/index.js:56:12)
    at runWithMerge (/example/node_modules/@putout/engine-runner/lib/index.js:65:5)
    at run (/example/node_modules/@putout/engine-runner/lib/index.js:54:12)
    at module.exports.runPlugins (/example/node_modules/@putout/engine-runner/lib/index.js:32:18)
    at transform (/example/node_modules/putout/lib/putout.js:99:20)
    at module.exports.findPlaces (/example/node_modules/putout/lib/putout.js:71:12)
    at Program (/example/node_modules/eslint-plugin-putout/lib/putout/index.js:54:32)
    at /example/node_modules/eslint/lib/linter/safe-emitter.js:45:58
    at Array.forEach (<anonymous>)
    at Object.emit (/example/node_modules/eslint/lib/linter/safe-emitter.js:45:38)
    at NodeEventGenerator.applySelector (/example/node_modules/eslint/lib/linter/node-event-generator.js:293:26)
    at NodeEventGenerator.applySelectors (/example/node_modules/eslint/lib/linter/node-event-generator.js:322:22)
    at NodeEventGenerator.enterNode (/example/node_modules/eslint/lib/linter/node-event-generator.js:336:14)
    at CodePathAnalyzer.enterNode (/example/node_modules/eslint/lib/linter/code-path-analysis/code-path-analyzer.js:711:23)
    at /example/node_modules/eslint/lib/linter/linter.js:960:32
    at Array.forEach (<anonymous>)
    at runRules (/example/node_modules/eslint/lib/linter/linter.js:955:15)
    at Linter._verifyWithoutProcessors (/example/node_modules/eslint/lib/linter/linter.js:1181:31)
    at Linter._verifyWithoutProcessors (/example/node_modules/eslint-plugin-eslint-comments/lib/utils/patch.js:181:42)
    at Linter._verifyWithConfigArray (/example/node_modules/eslint/lib/linter/linter.js:1280:21)
    at Linter.verify (/example/node_modules/eslint/lib/linter/linter.js:1235:25)
    at Linter.verifyAndFix (/example/node_modules/eslint/lib/linter/linter.js:1428:29)
    at verifyText (/example/node_modules/eslint/lib/cli-engine/cli-engine.js:234:48)
    at CLIEngine.executeOnFiles (/example/node_modules/eslint/lib/cli-engine/cli-engine.js:802:28)
    at ESLint.lintFiles (/example/node_modules/eslint/lib/eslint/eslint.js:563:23)
    at Object.execute (/example/node_modules/eslint/lib/cli.js:299:36)
    at main (/example/node_modules/eslint/bin/eslint.js:132:52)
    at Object.<anonymous> (/example/node_modules/eslint/bin/eslint.js:136:2)
    at Module._compile (internal/modules/cjs/loader.js:1085:14)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1114:10)
    at Module.load (internal/modules/cjs/loader.js:950:32)
    at Function.Module._load (internal/modules/cjs/loader.js:790:14)
    at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:76:12)
    at internal/main/run_main_module.js:17:47
```

**Comment by coderaiser** [comment]
Thank you! Fixed with https://github.com/coderaiser/estree-to-babel/commit/b780f6606e92495f828550d4417b93927dd4e9af. Landed in `estree-to-babel@4.2.0` 🎉 . Is it works for you?

**Comment by EvgenyOrekhov** [comment]
It works now, thank you!
